data/json/itemgroups/food_service.json

```
@@ -976,5 +976,50 @@
       { "item": "stout", "prob": 10, "container-item": "keg" },
       { "item": "imperial_stout", "prob": 5, "container-item": "keg" }
     ]
+  },
+  {
+    "type": "item_group",
+    "id": "butcher_raw_meat",
+    "items": [
+      [ "stomach_larget", 2 ],
```

```suggestion
      [ "stomach_large", 2 ],
```

Whoops, sorry, looks like a typo crept in.